lib/kubernetestoken/token_validator_test.go

```diff
@@ -144,8 +146,12 @@ func TestIDTokenValidator_Validate(t *testing.T) {
 					},
 				},
 			},
-			kubeVersion:   &boundTokenKubernetesVersion,
-			expectedError: trace.BadParameter("legacy SA tokens are not accepted as kubernetes version 1.22 supports bound tokens"),
+			kubeVersion: &boundTokenKubernetesVersion,
+			expectedError: trace.BadParameter(
+				fmt.Sprintf(
+					"legacy SA tokens are not accepted as kubernetes version %s supports bound tokens",
+					boundTokenKubernetesVersion.GitVersion),
```

Comment:
nit: does this not work without fmt?
```suggestion
			expectedError: trace.BadParameter(
				"legacy SA tokens are not accepted as kubernetes version %s supports bound tokens",
				boundTokenKubernetesVersion.GitVersion),
```